spec/models/purl_fetcher/record_deletes_spec.rb

```diff
@@ -13,8 +13,8 @@
         described_class.new(druid: 'abc123')
       end
       it 'enqueue a delete job' do
-        expect(Delayed::Job).to receive(:enqueue).with(DiscoveryDispatcher::IndexingJob.new('delete', 'abc123', 'searchworkspreview'))
-        expect(Delayed::Job).to receive(:enqueue).with(DiscoveryDispatcher::IndexingJob.new('delete', 'abc123', 'searchworks'))
+        expect(Delayed::Job).to receive(:enqueue).with(DiscoveryDispatcher::DeleteFromAllIndexesJob.new('delete', 'abc123', 'searchworkspreview'))
+        expect(Delayed::Job).to receive(:enqueue).with(DiscoveryDispatcher::DeleteFromAllIndexesJob.new('delete', 'abc123', 'searchworks'))
         subject.enqueue
```

Comment:
i find the name `DeleteFromAll...` combined with the subtarget confusing. i would expect a single expect for the DeleteFromAll case (i.e., no subtargets).